src/style.css

```diff
@@ -1,3 +1,64 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  height: 60px;
+  align-items: center;
+}
+
+.logo {
+  display: inline-block;
+  height: 40px;
+  width: 40px;
+  margin-left: 50px;
+  margin-top: 15px;
+  margin-bottom: 10px;
+  text-decoration: none;
+  text-align: center;
+}
+
+.nav_link {
+  display: block;
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  margin-left: 20px;
+  padding: 22.5px 10px;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+  margin-right: 50px;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  /* I don't now how I can do this */
+  content: '';
+  width: 100%;
+  height: 10px;
+  background-color: #00acdc;
+  border-radius: 30px;
+  text-decoration: underline;
+}
+
+.nav_link:hover {
+  color: #00acdc;
+}
+
+.nav_linki {
+  color: #00acdc;
+}
```

Comment:
The `.nav_linki` class appears to be redundant since it only changes the color to `#00acdc`, which is already handled by the `.nav_link:hover` and `.is-active` classes. Consider removing this class to simplify the CSS.